src/ToeplitzMatrices.jl

```diff
@@ -2,8 +2,9 @@ module ToeplitzMatrices
 # import StatsBase: levinson!, levinson
 import DSP: conv
 
-import Base: adjoint, convert, transpose, size, getindex, similar, copy, getproperty, inv, sqrt, copyto!, reverse, conj, zero, fill!, checkbounds, real, imag, isfinite, DimsInteger, iszero
+import Base: adjoint, convert, transpose, size, getindex, similar, copy, getproperty, inv, sqrt, copyto!, reverse, conj, zero, fill!, checkbounds, real, isfinite, DimsInteger, _all
```

Comment:
Why overload `_all` and not just `all`?

Reply:
Because `_all` is lower? I'm not quite sure how to choose.